src/deadline/houdini_submitter/python/deadline_cloud_for_houdini/submitter.py

```diff
@@ -332,18 +370,19 @@ def _get_job_template(rop: hou.Node) -> dict[str, Any]:
                 },
             },
         }
+
         if "dependency_names" in node:
             deps = [{"dependsOn": d} for d in node["dependency_names"]]
             step["dependencies"] = deps
         steps.append(step)
     job_template = {
         "specificationVersion": "jobtemplate-2023-09",
-        "name": job_name,
+        "name": rop.parm("name").evalAsString(),
+        "description": rop.parm("description").evalAsString() or None,
```

Comment:
Does this end up with `null` in the template? Is the behaviour different in the service if a description is `null` vs not there vs an empty string?

Reply:
It does add `null`s. They service accepts them. They are treated the same as if the key were not present. Empty strings are rejected though.

Reply:
Sounds good, just wanted to double check. Thanks!